openrtb_ext/bidders.go

```diff
@@ -37,6 +37,7 @@ const (
 	BidderAppnexus         BidderName = "appnexus"
 	BidderAdoppler         BidderName = "adoppler"
 	BidderBeachfront       BidderName = "beachfront"
+	BidderBeintoo          BidderName = "beintoo"
 	BidderBrightroll       BidderName = "brightroll"
 	BidderConsumable       BidderName = "consumable"
```

Comment:
This line is still missing in `openrtb_ext/bidders.go`:
```
 93   var BidderMap = map[string]BidderName{
 94       "33across":          Bidder33Across,
 95       "adform":            BidderAdform,
 96       "adgeneration":      BidderAdgeneration,
 97       "adkernel":          BidderAdkernel,
 98       "adkernelAdn":       BidderAdkernelAdn,
 99       "admixer":           BidderAdmixer,
100       "adpone":            BidderAdpone,
101       "adtelligent":       BidderAdtelligent,
102       "advangelists":      BidderAdvangelists,
103       "applogy":           BidderApplogy,
104       "appnexus":          BidderAppnexus,
105       "adoppler":          BidderAdoppler,
106       "beachfront":        BidderBeachfront,
    +     "beintoo":           BidderBeintoo,
107       "brightroll":        BidderBrightroll,
openrtb_ext/bidders.go
```

Reply:
sorry :-)